Parser/DelegateTokenFactory.php

```diff
@@ -28,8 +28,8 @@ class DelegateTokenFactory implements TokenFactoryInterface
     /**
      * DelegateTokenFactory constructor.
      *
-     * @param callable (TokenFactoryInterface $factory, array $params) $function
-     * @param TokenFactoryInterface|null                               $factory
+     * @param callable (TokenFactoryInterface, array $params): Token $function
```

Comment:
Explanation: phpstan requires callable phpdoc type to always state its return value, otherwise it fails with rather cryptic error about not recognizing `(` as valid token inside phpdoc